src/protected-api-access-protected-api/Program.cs

```diff
@@ -0,0 +1,27 @@
+// <ms_docref_import_types>
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Identity.Web;
+// </ms_docref_import_types>
+
+// <ms_docref_add_msal>
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddMicrosoftIdentityWebApi(builder.Configuration.GetSection("AzureAd"))
+                .EnableTokenAcquisitionToCallDownstreamApi()
+                .AddInMemoryTokenCaches()
+                .AddDownstreamWebApi("DownstreamApi", builder.Configuration.GetSection("DownstreamApi"));

```

Comment:
Any chance we can name this more meaningful?  Like `GraphApi` instead?

Reply:
good call @ckittel 

done | addressed [b40f3](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/13/commits/b40f3d67a1595d8fc5059a859f5f5d7ef5b03842)